cmd/ftl/cmd_subscription.go

```diff
@@ -0,0 +1,5 @@
+package main
```

Comment:
Let's call the top level command `pubsub`? So maybe `ftl pubsub subscription reset`? A little verbose, but I think we'll likely want more pubsub related commands, and they're not all subscriptions.